docs/existing-project/update-atri.md

```diff
@@ -63,6 +66,21 @@ rm -rf atri_app
 rm controllers/server.py
 ```
 
+</TabItem>
+<TabItem value="conda" label="Windows">
+
+```shell
+cd project_dir
```

Comment:
Replace `project_dir` with `my_app` since this is the name of the directory in this example.